src/services/book.service.ts

```
@@ -0,0 +1,275 @@
+
+            const fileExtension = path.extname(coverFile.originalname);
+            const fileName = `${Date.now()}-${Math.round(Math.random() * 1E9)}${fileExtension}`;
+            const filePath = path.join(uploadDir, fileName);
+
+            fs.writeFileSync(filePath, coverFile.buffer);
+            coverPath = `/images/covers/${fileName}`;
+
+            // Supprime l'ancienne couverture pour éviter l'accumulation de fichiers inutilisés
+            const oldBook = await getBook(id);
+            if (oldBook && oldBook.cover) {
+                const oldCoverPath = path.join(__dirname, '../../public', oldBook.cover);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)